src/components/buttons/ScrollableButton/hooks/useScrollHandler.ts

```diff
@@ -0,0 +1,19 @@
+import { useState } from 'react';
+import { NativeScrollEvent, NativeSyntheticEvent } from 'react-native';
+
+function useScrollHanlder() {
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  const handleOnScroll = (
+    scrollEvent: NativeSyntheticEvent<NativeScrollEvent>
+  ) => {
+    setScrollPosition(scrollEvent?.nativeEvent?.contentOffset.y);
+  };
+
+  return {
+    scrollPosition,
+    handleOnScroll,
+  };
+}
+
+export default useScrollHanlder;
```

Comment:
we could move this hook to `hooks` folder

Reply:
I think its ok to have it here since the usage of this hook is only for the ScrollableButton